### PR TITLE
Add scenario

### PR DESCRIPTION
**Pull request by invoceusse**
en-12
en-13

### CODE REVIEW

File: coops/emerald_island_scavenger_hunt.tex [hunk 1]
```diff
@@ -0,0 +1,115 @@
+% !TeX spellcheck = en_US
+\addscenariosection{1}{Cooperative scenario}{Emerald Island - Scavenger Hunt}{\images/logistics.png}
```
Comment (tomaas-zeman) [inline coops/emerald_island_scavenger_hunt.tex:2]:
```suggestion
\addscenariosection{2}{Cooperative scenario}{Emerald Island - Scavenger Hunt}{\images/logistics.png}
```

File: coops/emerald_island_scavenger_hunt.tex [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    \item You can give units, even outside your turn, to another player if one of your heroes are :
+    \begin{itemize}
+        \item In the same tile. 
+        \item In their castle.
+        \item Next to it.
+    \end{itemize}
+    \item If you have no unit in your unit deck (after a battle or if you not keep at least one unit) return all your unit to your faction's unit (even if other player are one or more of your unit).
+    \item When on another player's castle, you may recruit or reinforce their units, for them or for yourself (You may only do so if they have built the corresponding dwelling/citadel) if the other player are agreed with that.
+    \item for the last mine/settlement battle add a gold neutral unit. (you have unlimited turns for this battle.) You can't use expert diplomacy to skip this battle!
+    \item the first time you reach the level IV, gain your second specialty.
```
Comment (tomaas-zeman) [inline coops/emerald_island_scavenger_hunt.tex:82]:
```suggestion
    \item For the last mine/settlement battle add a gold neutral unit. (you have unlimited turns for this battle.) You can't use expert diplomacy to skip this battle!
    \item The first time you reach the level IV, gain your second specialty.
```

File: coops/emerald_island_scavenger_hunt.tex [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+\begin{itemize}
+    \item cannot build gold dwelling.
+    \item You can give units, even outside your turn, to another player if one of your heroes are :
+    \begin{itemize}
+        \item In the same tile. 
+        \item In their castle.
+        \item Next to it.
+    \end{itemize}
+    \item If you have no unit in your unit deck (after a battle or if you not keep at least one unit) return all your unit to your faction's unit (even if other player are one or more of your unit).
+    \item When on another player's castle, you may recruit or reinforce their units, for them or for yourself (You may only do so if they have built the corresponding dwelling/citadel) if the other player are agreed with that.
```
Comment (tomaas-zeman) [inline coops/emerald_island_scavenger_hunt.tex:80]:
```suggestion
    \item When on another player's castle, you may recruit or reinforce their units, for them or for yourself if the other player are agreed with that. You may only do so if they have built the corresponding dwelling/citadel.
```

File: coops/emerald_island_scavenger_hunt.tex [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+
+\textbf{5$^{th}$ Rounds:}
+\begin{itemize}
+  \item gain again the result of one field with black cube.
+\end{itemize}
+
+\subsection*{\MakeUppercase{Additional Rules}}
+\begin{itemize}
+    \item cannot build gold dwelling.
+    \item You can give units, even outside your turn, to another player if one of your heroes are :
```
Comment (tomaas-zeman) [inline coops/emerald_island_scavenger_hunt.tex:73]:
```suggestion
    \item You can give units, even outside your turn, to another player if one of your heroes are:
```

File: coops/emerald_island_scavenger_hunt.tex [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+\begin{minipage}{0.4\paperwidth}
+  \centering
+  \includegraphics[width=0.38\paperwidth]{\_assets/maps/emerald-3.png}
+  \captionof{figure}{3 players}
+\end{minipage}
+\begin{minipage}{0.4\paperwidth}
+  \centering
+  \includegraphics[width=0.38\paperwidth]{\_assets/maps/emerald-4.png}
+  \captionof{figure}{4 players}
+\end{minipage}
```
Comment (tomaas-zeman) [inline coops/emerald_island_scavenger_hunt.tex:115]:
```suggestion
\vspace{3em}

\begin{center}
  \includegraphics[width=0.2\paperwidth]{\_assets/maps/emerald-1.png}
  \captionof{figure}{1-PLAYER SCENARIO}
  \vspace{3em}
  \includegraphics[width=0.3\paperwidth]{\_assets/maps/emerald-2.png}
  \captionof{figure}{2-PLAYER SCENARIO}
  \vspace{3em}
  \includegraphics[width=0.3\paperwidth]{\_assets/maps/emerald-3.png}
  \captionof{figure}{3-PLAYER SCENARIO}
  \vspace{3em}
  \includegraphics[width=0.3\paperwidth]{\_assets/maps/emerald-4.png}
  \captionof{figure}{4-PLAYER SCENARIO}
\end{center}

\end{multicols*}
```

File: coops/emerald_island_scavenger_hunt.tex [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+\subsection*{\MakeUppercase{Additional Rules}}
+\begin{itemize}
+    \item cannot build gold dwelling.
+    \item You can give units, even outside your turn, to another player if one of your heroes are :
+    \begin{itemize}
+        \item In the same tile. 
+        \item In their castle.
+        \item Next to it.
+    \end{itemize}
+    \item If you have no unit in your unit deck (after a battle or if you not keep at least one unit) return all your unit to your faction's unit (even if other player are one or more of your unit).
```
Comment (tomaas-zeman) [inline coops/emerald_island_scavenger_hunt.tex:79]:
```suggestion
    \item If you have no unit in your unit deck (after a battle or if you don't keep at least one unit) return all your unit to your faction's unit (even if other player are one or more of your unit).
```

Comment (tomaas-zeman) [inline coops/emerald_island_scavenger_hunt.tex:79]: Could you rephrase this? I don't fully understand what is meant by it.

File: coops/emerald_island_scavenger_hunt.tex [hunk 7]
```diff
@@ -0,0 +1,115 @@
...
+\subsection*{\MakeUppercase{Timed Events}}
+
+\textbf{5$^{th}$ Rounds:}
+\begin{itemize}
+  \item gain again the result of one field with black cube.
+\end{itemize}
+
+\subsection*{\MakeUppercase{Additional Rules}}
+\begin{itemize}
+    \item cannot build gold dwelling.
```
Comment (tomaas-zeman) [inline coops/emerald_island_scavenger_hunt.tex:72]:
```suggestion
    \item You can't build gold dwelling.
```

File: coops/emerald_island_scavenger_hunt.tex [hunk 8]
```diff
@@ -0,0 +1,115 @@
...
+
+\subsection*{\MakeUppercase{Defeat Conditions}}
+
+There are unflagged mines or settlements left in map at the end of Round 8.
+
+\subsection*{\MakeUppercase{Timed Events}}
+
+\textbf{5$^{th}$ Rounds:}
+\begin{itemize}
+  \item gain again the result of one field with black cube.
```
Comment (tomaas-zeman) [inline coops/emerald_island_scavenger_hunt.tex:67]:
```suggestion
  \item Gain again the result of one field with black cube.
```

File: coops/emerald_island_scavenger_hunt.tex [hunk 9]
```diff
@@ -0,0 +1,115 @@
...
+
+Flag all mines and settlements. (Don't forget the mine of building materials in tile I, and each tile II-III are one mine (or settlements) with III fight)
+
+\subsection*{\MakeUppercase{Defeat Conditions}}
+
+There are unflagged mines or settlements left in map at the end of Round 8.
+
+\subsection*{\MakeUppercase{Timed Events}}
+
+\textbf{5$^{th}$ Rounds:}
```
Comment (tomaas-zeman) [inline coops/emerald_island_scavenger_hunt.tex:65]:
```suggestion
\textbf{5$^{th}$ Round:}
```

File: coops/emerald_island_scavenger_hunt.tex [hunk 10]
```diff
@@ -0,0 +1,115 @@
...
+\end{itemize}
+
+\subsection*{\MakeUppercase{Additional Rules}}
+\begin{itemize}
+    \item cannot build gold dwelling.
+    \item You can give units, even outside your turn, to another player if one of your heroes are :
+    \begin{itemize}
+        \item In the same tile. 
+        \item In their castle.
+        \item Next to it.
```
Comment (tomaas-zeman) [inline coops/emerald_island_scavenger_hunt.tex:77]: Next to what? :)

File: coops/emerald_island_scavenger_hunt.tex [hunk 11]
```diff
@@ -0,0 +1,115 @@
...
+\begin{itemize}
+  \item gain again the result of one field with black cube.
+\end{itemize}
+
+\subsection*{\MakeUppercase{Additional Rules}}
+\begin{itemize}
+    \item cannot build gold dwelling.
+    \item You can give units, even outside your turn, to another player if one of your heroes are :
+    \begin{itemize}
+        \item In the same tile.
```
Comment (tomaas-zeman) [inline coops/emerald_island_scavenger_hunt.tex:75]: Do you mean in the same map tile?

File: coops/emerald_island_scavenger_hunt.tex (hunk 3 above)
Comment (qwrtln) [reply coops/emerald_island_scavenger_hunt.tex:80]: castle -> town

Comment (qwrtln) [reply coops/emerald_island_scavenger_hunt.tex:80]: How does reinforcing their units work? They have a "few", you pay, and they receive a "pack"? Who flips the hashtag token?

File: coops/emerald_island_scavenger_hunt.tex (hunk 11 above)
Comment (invoceusse) [reply coops/emerald_island_scavenger_hunt.tex:75]: Yep

File: coops/emerald_island_scavenger_hunt.tex (hunk 10 above)
Comment (invoceusse) [reply coops/emerald_island_scavenger_hunt.tex:77]: Next to another heroes of player (main or secondary)

File: coops/emerald_island_scavenger_hunt.tex (hunk 3 above)
Comment (invoceusse) [reply coops/emerald_island_scavenger_hunt.tex:80]: > How does reinforcing their units work? They have a "few", you pay, and they receive a "pack"? Who flips the hashtag token?

- If player A (dungeon faction) has Troglodytes and citadel, player B can go in town A, flip player B # and reinforce Troglodytes A

- If player A (dungeon faction) has Troglodytes and no citadel, but player B has citadel, player B can't reinforce Troglodytes

- If player B (not dungeon faction) received Troglodytes from player A and player A has this citadel, Player can reinforce Troglodytes.


So need factions' citadel and it's the player who pay the unit who flip this token

File: coops/emerald_island_scavenger_hunt.tex (hunk 10 above)
Comment (tomaas-zeman) [reply coops/emerald_island_scavenger_hunt.tex:77]: I would prefer being specific here. While reading the list of items and seeing "in their town" and "next to it" as a next point feels like "it" is the "town"

Comment (invoceusse) [reply coops/emerald_island_scavenger_hunt.tex:77]: 👌

**Review by tomaas-zeman — APPROVED** [review]

**Review by qwrtln — APPROVED** [review]

**Comment by qwrtln** [comment]
The spacing below the scenario header looks a bit off, can it be fixed?
